tasks/tests/test_view_all_tasks.py

```diff
@@ -32,3 +34,24 @@ def test_authenticated_user_tasks_view(self, client, test_db, employee_flag):
             assert all(task.assignee == user for task in response.context['tasks'])
         else:
             assert all(task.assignee.team == user.team for task in response.context['tasks'])
+
+    def test_filter_task_by_priority(self, client, test_db):
+        user = test_db[1][0]
+        client.login(username=user.user.username, password=DEFAULT_VALID_PASSWORD)
+        response = client.post('/tasks/', data={'priority': 'Low'})
+        assert all(task.priority == Priority.LOW for task in response.context['tasks'])
+
+    def test_filter_task_by_priority_randomized(self, client, test_db):
```

Comment:
Usually, when you run your tests you want to know the correct values that you pass and get.
so the randomized attitude is not recommended.
also, take into account that it could be that you will get the same values a few times which is waste of time and resources.
I would recommend using the parameterize instead.

Reply:
that's why it runs only 10 times

Reply:
Could you please elaborate on how it is better if you run the same test 10 times(in the worst case) instead of 10 different tests